lib/oli/delivery/metrics.ex

```diff
@@ -55,7 +75,11 @@ defmodule Oli.Delivery.Metrics do
   def progress_across(section_id, container_ids, user_id) do
     query =
       ContainedPage
-      |> join(:left, [cp], ra in ResourceAccess, on: cp.page_id == ra.resource_id and cp.section_id == ra.section_id and ra.user_id == ^user_id)
+      |> join(:left, [cp], ra in ResourceAccess,
+        on:
+          cp.page_id == ra.resource_id and cp.section_id == ra.section_id and
+            ra.user_id == ^user_id
+      )
```

Comment:
Same as above

Reply:
In this case left join is needed because we do want to get the nil entry in order to calculate the total count over the ContainedPage table